### PR TITLE
Add new custom sauce command: jankinessCheck

### PR DESCRIPTION
**Pull request by christian-bromann**
## Proposed changes

Sauce Labs will soon provide a new command that allows to check how smooth the website acted while interactive with it. It gives a sense of the so called "jankiness" of the page. The command is already available for beta and will be soon released to all performance user.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: packages/wdio-protocols/protocols/saucelabs.json [hunk 1]
```diff
@@ -199,5 +209,25 @@
         "description": "Returns true if any metrics have a regression."
       }
     }
+  },
+  "/session/:sessionId/sauce/ondemand/performance/scroll": {
+    "POST": {
+      "command": "jankinessCheck",
+      "description": "Perform a scroll test that evaluates the jankiness of the application.",
+      "ref": "https://wiki.saucelabs.com/display/DOCS/Custom+Sauce+Labs+WebDriver+Extensions+for+Network+and+Log+Commands",
```
Comment (mgrybyk) [inline packages/wdio-protocols/protocols/saucelabs.json]: Can't see nothing here about scroll performance. Is it going to be added soon?

File: packages/wdio-protocols/protocols/saucelabs.json [hunk 2]
```diff
@@ -199,5 +209,25 @@
         "description": "Returns true if any metrics have a regression."
       }
     }
+  },
+  "/session/:sessionId/sauce/ondemand/performance/scroll": {
+    "POST": {
+      "command": "jankinessCheck",
```
Comment (mgrybyk) [inline packages/wdio-protocols/protocols/saucelabs.json]: shouldn't command name be something like getScrollPerformance ?

File: packages/wdio-protocols/protocols/saucelabs.json (hunk 1 above)
Comment (christian-bromann) [reply packages/wdio-protocols/protocols/saucelabs.json]: yes

File: packages/wdio-protocols/protocols/saucelabs.json (hunk 2 above)
Comment (christian-bromann) [reply packages/wdio-protocols/protocols/saucelabs.json]: We are not sure with the naming yet, jankiness can also be captured without scrolling. This command just does scrolling as one method. Overall the metrics that are return give information about how smooth the app behaved meaning how janky it was.

**Pull request merged**